### DISCUSSION TITLE
How to run it else than command method

### DISCUSSION THREAD

**Discussion by AffanAhmedUsmani** [nerdyrodent/VQGAN-CLIP#139] [Q&A]
I  was here to ask this that how can I run this from another py file in the same directory like without using it in cmd like python generate.py -p " some text " I would like to import generate.py then how would I say -p "some argument  " help would be highly appreciated

**Comment by manafoo** [comment]
i think you can create a .bat file to launch it , but you need to know what to type , i've created a video on youtube on how to use google colab with ur local machine , you will get the benefit of the interface without having to ype into cmd everytime
